### PR TITLE
:bug: fix: Copy indexer.toml in dockerfile

### PR DESCRIPTION
**Pull request by roninjin10**
- Copy default foundry.toml file into docker build so it can be used

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #7089** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7089" target="_blank">Graphite</a> 
    * **PR #7090** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7090" target="_blank">Graphite</a> 
      * **PR #7091** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7091" target="_blank">Graphite</a> 
        * **PR #7088** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7088" target="_blank">Graphite</a>  👈
          * **PR #7092** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7092" target="_blank">Graphite</a> 
            * **PR #7257** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7257" target="_blank">Graphite</a> 
            * **PR #7247** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7247" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/7088?utm_source=stack-comment).

File: indexer/docker-compose.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ services:
     environment:
       - INDEXER_RPC_URL_L1=$INDEXER_RPC_URL_L1
       - INDEXER_RPC_URL_L2=$INDEXER_RPC_URL_L2
-      - INDEXER_CONFIG=/indexer/indexer.toml
+      - INDEXER_CONFIG=/configs/indexer.toml
```
Comment (roninjin10) [inline indexer/docker-compose.yml:28]: Could remove the volumes: below since we aren't using it anymore

**Comment by hamdiallam** [comment]
I think you have a typo in the PR title (foundry) lol

**Review by hamdiallam — COMMENTED** [review]
Is it right to make the default toml configuration directly apart of the docker image?

Having the volume there makes it explicit that the consumer of the image needs to supply the right config

**Comment by roninjin10** [comment]
@hamdiallam it's still explicit in these ways
1. The docs tell you to use a toml
2. The toml in the image isn't used by default you would need to explicitly use it
In future I'd imagine we just go ahead and provide default tomls for all preset chains

**Review by hamdiallam — APPROVED** [review]

**Comment by roninjin10** [comment]
lol oops merged into wrong branch

**Pull request merged**